source/OrderedTypes/ZornsLemma.lagda

```diff
@@ -0,0 +1,375 @@
+Kelton OBrien, 31 May 2024
+
+A proof that the Axiom of Choice implies Zorn's Lemma, as well as relevant definitions.
+\begin{code}
+
+{-# OPTIONS --safe --without-K --exact-split #-}
+
+open import MLTT.Spartan
```

Comment:
Maybe sort the imports? You can do this by marking the block in `emacs` and then running `sort-lines`.